src/go/rpk/pkg/cli/cmd/cluster/config/reset.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"path/filepath"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	"gopkg.in/yaml.v2"
+)
+
+func newResetCommand(fs afero.Fs) *cobra.Command {
+	var configCacheFile string
+	cmd := &cobra.Command{
+		Use:   "reset [PROPERTY...]",
+		Short: "Forcibly clear a cluster configuration property on this node.",
+		Long: `Forcibly clear a cluster configuration property on this node.
+
+This command is not for general changes to cluster configuration: use this only
+when redpanda will not start due to a configuration issue.
```

Comment:
Is there a use case to use `reset` as a shortcut to `set` a value to its default if it was previously `set` to a custom value?

Reply:
No, definitely not (that is not what it does).  I'm actually kind of concerned about the risk of people misusing it that way.

Reply:
This question prompted me to think about the risks of a user mistaking the `reset` command for something more friendly than it is.  I've added a commit that renames it to `force-reset` and another commit that provides a friendly way to reset a property to a default by assigning an empty string with `set`.

Reply:
I wonder if this should be named `force-local-reset` just to be uber clear, but `force-reset` is clearer too.